srml/contract/src/account_db.rs

```diff
@@ -43,8 +44,44 @@ impl<T: Trait> Default for ChangeEntry<T> {
 
 pub type ChangeSet<T> = BTreeMap<<T as system::Trait>::AccountId, ChangeEntry<T>>;
 
+#[derive(Clone, Default)]
+pub struct AccountKeySpaceMapping<A: Ord> {
```

Comment:
It is a cache to avoid accessing subtrie data through direct db at every call to a subkey.
Storage got its own lru cache, so I am starting to wonder if it is totally usefull.
Could prove more usefull if it mapped account_id to `SubTrie` struct.

Reply:
> Storage got its own lru cache

What do you mean? I can't recall anything about that...

Reply:
Some LruCache usage into core/client/db/src/storage_cache.rs , but with a `AccountInfo` struct we probably do not want to go down to this and deserialize each time.